uint8 mission_type
